README.md

```diff
@@ -45,6 +48,12 @@ const logger = new Logify<CustomErrorType>({
     warn?: string,
     error?: string,
     fatal?: string,
+  },
+  // Storage configuration
+  storage?: {
```

Comment:
@janisievins sorry small typo i missed: the storage key shouldn't be followed by a `?` here as well

Reply:
Fixed the whole example in README. ✅